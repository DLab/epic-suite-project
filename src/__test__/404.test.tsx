import { Link, Icon } from "@chakra-ui/react";
import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";

import Page404 from "../pages/404";

describe("Home", () => {
    it("renders page correctly", () => {
        render(<Page404 />);
    });
    it("renders correctly", () => {
        const homePageLink = renderer
            .create(<Link href="/">Home Page</Link>)
            .toJSON();
        expect(homePageLink).toMatchSnapshot();
    });
});
