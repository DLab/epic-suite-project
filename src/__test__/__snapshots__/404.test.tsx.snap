// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<a
  className="chakra-link css-0"
  href="/"
>
  Home Page
</a>
`;
